src/app/views/teams/teams-view/TeamsList.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router";
+import url from "../../../utilities/url";
+import Input from "../../../components/Input";
+import SimpleSelectableList from "../../../components/simple-selectable-list/SimpleSelectableList";
+
+const propTypes = {
+    dispatch: PropTypes.func,
+    teams: PropTypes.object.isRequired,
+    loadTeams: PropTypes.func.isRequired,
+};
+
+const TeamsList = props => {
+    const { teams, loadTeams } = props;
+
+    const [isFetchingTeams, setIsFetchingTeams] = useState(true);
+    const [filterTerm, setFilterTerm] = useState("");
+    const [previewTeams, setPreviewTeams] = useState([]);
+
+    useEffect(() => {
+        loadTeams();
+    }, []);
+
+    useEffect(() => {
+        const listOfPreviewTeams = [];
+        teams?.groups?.forEach(team => {
```

Comment:
what is the shape of this teams object?

Reply:
we should not have `team={ groups: []}` but `groups=[]` for now. in the future works on optimising Florence we  will normalise the json object coming form api